sbol_utilities/sbol3_sbol2_conversion.py

```diff
@@ -266,7 +274,9 @@ def visit_sequence(self, seq3: sbol3.Sequence):
                         sbol3.SMILES_ENCODING: sbol2.SBOL_ENCODING_SMILES}
         encoding2 = encoding_map.get(seq3.encoding, seq3.encoding)
         # Make the Sequence object and add it to the document
-        seq2 = sbol2.Sequence(seq3.identity, seq3.elements, encoding=encoding2, version=self._sbol2_version(seq3))
+        seq2 = sbol2.Sequence(seq3.identity, seq3.elements, 
```

Comment:
Please don't change the line-wrapping standard on the project. Per the setup.py file at the top, this project uses `max-line-length = 119`.
There should not be any change to this line of code, since it's just a line-wrapping change.